crates/dbs-legacy-devices/Cargo.toml

```diff
@@ -16,6 +16,7 @@ log = "0.4.14"
 serde = { version = "1.0.27", features = ["derive", "rc"] }
 vm-superio = "0.5.0"
 vmm-sys-util = "0.9.0"
+byteorder = "1.4.3"
```

Comment:
Only introduce the dependency for aarch64?
```
[target.'cfg(target_arch = "aarch64")'.dependencies]
```

Reply:
Done, I have remove byteorder.